sql/views.sql

```diff
@@ -259,5 +259,124 @@ CREATE OR REPLACE VIEW timescaledb_information.data_node AS
     LEFT OUTER JOIN LATERAL @extschema@.data_node_hypertable_info(CASE WHEN s.node_up THEN s.node_name ELSE NULL END) size ON TRUE
   GROUP BY s.node_name, s.node_up, s.owner, s.options;
 
+-- chunks metadata view, shows information about the primary dimension column
+CREATE OR REPLACE VIEW  timescaledb_information.chunks
+AS
+SELECT 
+       hypertable_schema, hypertable_name,
+       schema_name as chunk_schema , chunk_name ,
+       primary_dimension, primary_dimension_type,
+       range_start, range_end,
+       integer_range_start as range_start_integer,
+       integer_range_end as range_end_integer,
+       is_compressed,
+       chunk_table_space as chunk_tablespace,
+       node_list
```

Comment:
It was proposed `data_nodes` in the design doc.

Reply:
thanks. fixed that.